<% provide(:title, "Error 422") %>

<section id="422">
<h1> Oops, we seem to have a glitch in our system. </h1>
<h2> You might try reloading the page in a little while or <%= link_to "contact us", contact_path %> if the problem persists. </h2>
</section>